client/reader/stats.js

```diff
@@ -96,6 +96,11 @@ export function recordTrack( eventName, eventProperties ) {
 		eventProperties = Object.assign( { subscription_count: subCount }, eventProperties );
 	}
 
+	if ( location === 'topic_page' && ! eventProperties.hasOwnProperty( 'tag' ) ) {
+		const tag = decodeURIComponent( window.location.pathname.split( '/tag/' ).pop() );
+		eventProperties = Object.assign( { tag: tag }, eventProperties );
```

Comment:
not sure why we keep making new objects instead of mutating here...but its the trend in this function